files/jsbarcode/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "JsBarcode",
+  "repo": "lindell/JsBarcode",
+  "files": {
+    "include": ["bin/browser/*.min.*", "bin/browser/barcodes/*.min.*"]
```

Comment:
This wont create a "barcodes" folder.
You should set the `basePath` parameter to `bin/browser/` and then do `*.min.* and `./barcodes/*.min.*`